app/models/users/notification_setting.rb

```diff
@@ -1,10 +1,22 @@
 module Users
   class NotificationSetting < ApplicationRecord
     self.table_name_prefix = "users_"
+
+    belongs_to :user, touch: true
+
     validates :email_digest_periodic, inclusion: { in: [true, false] }
+    validates :welcome_notifications, inclusion: { in: [true, false] }
```

Comment:
We can remove this validation. The boolean column is `null: false` so all values that aren't true or false will be rejected. And we're populating this column ourselves, there should be no need to be so cautious. See the link that was shared in our Slack not too long ago:

http://www.chrisrolle.com/en/blog/boolean-attribute-validation

Reply:
I totally just discovered this while trying to put bad data in the database! 💯